roaster-zephyr/roaster-zephyr-common/src/main/java/hu/icellmobilsoft/roaster/zephyr/common/client/api/ZephyrRestClient.java

```diff
@@ -57,6 +57,17 @@ public interface ZephyrRestClient {
     @Path("/testcases/{testCaseKey}")
     Response getTestCase(@PathParam("testCaseKey") String testCaseKey);
 
+    /**
+     * Return the test steps of the given test case
+     *
+     * @param testCaseKey
+     *            test case key to find
+     * @return response containing HTTP status {@literal 200} if the test case exists or {@literal 404} if not
+     */
+    @GET
+    @Path("/testcases/{testCaseKey}/teststeps")
+    Response getTestCaseSteps(@PathParam("testCaseKey") String testCaseKey);
```

Comment:
Fent is fura, hogyha tudjuk hogy mi a response, itt miért nem azt használjuk? És akkor rögtön feloldaná a rest  client, nem kéne ObjectMappert használni külön.

Reply:
Fel vehetünk rá taskot, egyben lenne jó akkor átírni, ha csak nincs valami oka annak hogy így lett kifejezetten megoldva...

Reply:
Fentebb azért vannak Response-ok, mert nem érdekel minket a body, csak, hogy létezik-e az adott objektum. Ha parse-olnánk a body-t (mellesleg feleslegesen), akkor sokkal gyakrabban törne el ez a modul. Mivel külső fél kezeli az api-t szerintem jobb minél kevésbé függnünk a változásoktól.